framegear/components/Frame/Frame.tsx

```diff
@@ -119,13 +128,16 @@ function FrameButton({
             fid: 0,
             hash: '0xthisisnotreal',
           },
-          inputText: '',
+          inputText,
           fid: 0,
           messageHash: '0xthisisnotreal',
           network: 0,
           timestamp: 0,
         });
-        setResults((prev) => [...prev, result]);
```

Comment:
we got an error on this because `postFrame` may return an undefined value, and only defined values should go in the results array.